plugins/nodes/src/js/components/NodesTableWrapper.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import graphqlObservable from "reactive-graphql";
+import gql from "graphql-tag";
+import { Observable, combineLatest } from "rxjs";
+import { map, switchMap } from "rxjs/operators";
+import { componentFromStream } from "@dcos/data-service";
+import { TableProps } from "@dcos/ui-kit/dist/packages/table/components/Table";
+
+import { NodeNetwork } from "../data/NodesNetworkClient";
+import { default as schema } from "../data/NodesResolver";
+import Node from "#SRC/js/structs/Node";
+
+interface TableWrapperProps {
+  data: any[];
+  children: React.ReactElement<TableProps>;
+}
+
+const getGraphQl = (privateIP: string) => {
+  return graphqlObservable<{ node: { network: NodeNetwork } }>(
+    gql`
+      query {
+        node(privateIP: $privateIP) {
+          network {
+            public_ips
+          }
+        }
+      }
+    `,
+    schema,
+    { privateIP }
+  ).pipe(map(response => response.data));
+};
+// Put in the same props as you would in the NodesTable
+const NodesTableWrapper = componentFromStream<TableWrapperProps>(props$ => {
+  const data$ = (props$ as Observable<TableWrapperProps>).pipe(
```

Comment:
We need the cast since props$ is a `Subscribable<TableWrapperProps>` we could omit the generic type on the function call? But I thought it might be a bad idea. If you have a better approach please let me know.